IoTCloudSDK/IoTCloudSDK/IoTCloudAPI.swift

```diff
@@ -7,7 +7,15 @@ import Foundation
 
 /** Class provides API of the IoTCloud. */
 public class IoTCloudAPI: NSObject, NSCoding {
-    
+
+    private static let SHARED_PREFERENCES_KEY_INSTANCE = "IoTCloudAPI_INSTANCE"
```

Comment:
shared preference is term of Android. Let us make it iOS like. What do you think? :)